source/common/quic/platform/quiche_flags_impl.cc

```diff
@@ -38,6 +38,10 @@ template <> constexpr bool maybeOverride<bool>(absl::string_view name, bool val)
     // Do not include 32-byte per-entry overhead while counting header size.
     return false;
   }
+  if (name == "quic_reloadable_flag_quic_act_upon_invalid_header") {
+    // This flag changes quiche behavior that's incompatible with current Envoy.
+    return false;
```

Comment:
Do we have an issue tracking this internally so we don't end up removing this flag until we've sorted out what to do in Envoy?

Reply:
I just filed one.